polynote-frontend/polynote/ui/component/search.ts

```diff
@@ -77,6 +78,20 @@ export class SearchModal extends Modal implements IDisposable {
             })
 
         }).disposeWith(this);
+
+        // in case something goes wrong while searching, display an error message and update the "searching..." message
+        ErrorStateHandler.get.view("serverErrors").addObserver((errors) => {
+            let message = "Something went wrong while searching" + (errors.length > 0 ? ":\n\n" : ", but we're unable to display the error.");
```

Comment:
If we get an update with empty errors, doesn't it mean the error state was cleared?

Reply:
oops, the server actually currently doesn't send any message if the parsingfailure goes away. let me see if I can fix that